src/context.ml

```diff
@@ -426,6 +428,23 @@ let install_prefix t =
   | Some x -> Path.absolute x
   | None   -> Path.parent t.ocaml_bin
 
+let install_ocaml_libdir t =
+  (* If ocamlfind is present, it has precedence over everything else. *)
+  match which t "ocamlfind" with
+  | Some fn ->
+    (Future.run_capture_line ~env:t.env Strict
+       (Path.to_string fn) ["printconf"; "destdir"]
+     >>| Path.absolute)
+  | None ->
+    (* If there no ocamlfind in the PATH, check if we have opam and assume a standard
+       opam setup *)
+    opam_config_var t "lib"
+    >>| function
+    | Some s -> Path.absolute s
+    | None ->
+      (* If neither opam neither ocamlfind are present use stdlib dir *)
+      t.stdlib_dir
```

Comment:
Not really, the current default is where OCaml is installed which seems fine to me, especially if we install the library files in the stdlib directory